src/SOS/SOS.Package/SOS.Package.csproj

```diff
@@ -0,0 +1,120 @@
+﻿<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" />
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <AssemblyName>Microsoft.NETCore.SOS</AssemblyName>
+    <Description>.NET Core SOS binaries</Description>
+    <PackageReleaseNotes>$(Description)</PackageReleaseNotes>
+    <IsPackable>true</IsPackable>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+    <SOSPackagePathPrefix>tools</SOSPackagePathPrefix>
+    <GalleryManifestName>$(ArtifactsPackagesDir)\GalleryManifest.xml</GalleryManifestName>
+    <BeforePack>GenerateGalleryZip</BeforePack>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="$(GalleryManifestName)" Pack="true" Visible="false">
+      <PackagePath>$(SOSPackagePathPrefix)</PackagePath>
+    </None>
+  </ItemGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)..\..\sos-packaging.props" />
```

Comment:
Giving them this will also make them have all the linux and OSX versions of SOS. Do we want the package to have all this?

Reply:
I look at this package as the SOS package that might be useful in other situations not just the Windows debugger package.

Reply:
For stand alone tools like dotnet-sos I got this, given they were self contained. Given we don't have a good restore logic per rid until now, I am ok with this, but at some point we should revisit.